src/features/canvassAssignments/utils/getAreaData.spec.ts

```diff
@@ -0,0 +1,655 @@
+import { Household } from '../types';
+import getAreasData from './getAreaData';
+
+describe('getAreasData()', () => {
```

Comment:
The function is in a module called `getAreaData`, but it's imported as `getAreasData`. I think the singular name makes more sense. No need to change it now, but maybe in your next PR?